paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -109,6 +109,56 @@ static void AppendActivation(const framework::ExecutionContext& ctx,
   }
 }
 
+static void SetOutMemDescWithLogicalLayoutFusesSupport(
+    const framework::ExecutionContext& ctx,
+    phi::DenseTensor* out,
+    const dnnl::memory::desc& out_md) {
+  out->set_mem_desc(out_md);
+  out->Resize(phi::make_ddim(out_md.dims()));
+}
```

Comment:
These lines will be added in https://github.com/PaddlePaddle/Paddle/pull/47391 PR and shouldn't be here, because it will create conflicts

Reply:
same above